pom.xml

```diff
@@ -13,20 +13,34 @@
         <jda.version>4.0.0_39</jda.version>
         <lavaplayer.version>1.3.32</lavaplayer.version>
         <lavalink.client.version>f9ab37f</lavalink.client.version>
+        <kotlin.version>1.3.61</kotlin.version>
     </properties>
 
     <build>
         <finalName>CascadeBot</finalName>
-        <sourceDirectory>src/main/java</sourceDirectory>
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.8.0</version>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-maven-plugin</artifactId>
+                <version>${kotlin.version}</version>
+                <executions>
+                    <execution>
+                        <id>compile</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>test-compile</id>
+                        <phase>test-compile</phase>
+                        <goals>
+                            <goal>test-compile</goal>
+                        </goals>
+                    </execution>
+                </executions>
                 <configuration>
-                    <source>11</source>
-                    <target>11</target>
-                    <release>11</release>
+                    <jvmTarget>1.8</jvmTarget>
```

Comment:
1.8? We run Java 11 👀

Reply:
I figured it wouldn't make much of a difference just for testing if jenkins was gonna work or not, I'll change it